src/ibex_bluesky_core/run_engine.py

```diff
@@ -65,4 +67,7 @@ def get_run_engine() -> RunEngine:
     )
     RE.record_interruptions = True
 
+    log_callback = DocLoggingCallback()
+    RE.subscribe(log_callback)  # Uses ID = 0 for DocLoggingCallback
```

Comment:
I thought it was the RE that would tell you what your callback ID is (which might change if in future we add some other callback before this one...)

Not that we actually _care_ for this - I think the ID is only needed if you want to unsubscribe a callback, which we don't want to for this one.